src/physics/em/detail/BetheHeitler.cu

```diff
@@ -0,0 +1,82 @@
+//---------------------------------*-CUDA-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file BetheHeitler.cu
+//---------------------------------------------------------------------------//
+#include "BetheHeitler.hh"
+
+#include "base/KernelParamCalculator.cuda.hh"
+#include "physics/base/SecondaryAllocatorView.hh"
+#include "physics/base/ModelInterface.hh"
+#include "physics/base/ParticleTrackView.hh"
+#include "physics/base/PhysicsTrackView.hh"
+#include "random/cuda/RngEngine.hh"
+#include "BetheHeitlerInteractor.hh"
+
+namespace celeritas
+{
+namespace detail
+{
+namespace
+{
+//---------------------------------------------------------------------------//
+// KERNELS
+//---------------------------------------------------------------------------//
+/*!
+ * Interact using the Bethe-Heitler model on applicable tracks.
+ */
+__global__ void bethe_heitler_interact_kernel(const BetheHeitlerPointers  bh,
+                                              const ModelInteractPointers ptrs)
+{
+    auto tid = celeritas::KernelParamCalculator::thread_id();
+    if (tid.get() >= ptrs.states.size())
+        return;
+
+    SecondaryAllocatorView allocate_secondaries(ptrs.secondaries);
+    ParticleTrackView particle(ptrs.params.particle, ptrs.states.particle, tid);
+
+    PhysicsTrackView physics(ptrs.params.physics,
+                             ptrs.states.physics,
+                             particle.def_id(),
+                             MaterialDefId{},
+                             tid);
+
+    // This interaction only applies if the Bethe-Heitler model was selected
+    if (physics.model_id() != bh.model_id)
+        return;
+
+    BetheHeitlerInteractor interact(
+        bh, particle, ptrs.states.direction[tid.get()], allocate_secondaries);
```

Comment:
There's a bit of work to do in this class -- you'll need to instantiate the material view, (for now) add a check that there's only one element in the current material, and then pass the element view into the interactor.  I think this didn't get caught because these new files haven't been added to the `src/CMakeLists.txt`.